cmd/geth/chaincmd.go

```diff
@@ -206,7 +206,7 @@ func initGenesis(ctx *cli.Context) error {
 	defer stack.Close()
 
 	for _, name := range []string{"chaindata", "lightchaindata"} {
-		chaindb, err := stack.OpenDatabase(name, 0, 0, "", false)
```

Comment:
In this case, the freezer is always initialized with the default path `<DATADIR>/chaindata/ancient`.
Although we never offer the ancient flag for customizing the ancient path though(it should be fixed).

Reply:
Yep, I'll wait for https://github.com/ethereum/go-ethereum/pull/24698 to get merged and take ancient path from the flag added there

Reply:
Ping, #24698 merged

Reply:
Updated